### ISSUE TITLE
508 Issues - From #2207 (View Monitoring Qualifications Data)

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#2266]
Must resolve:

- Qualifications is a visual heading but the html has it as a span tag
- The name of the data table is generic: aria-label="Data Table". Should be labeled by the qualification text.

**Comment by dionolympiaCVP** [comment]
First requirement (span elements converted to h3) was completed in [Ticket 2399](https://app.zenhub.com/workspaces/emissioners-scrum-board-5f36cd263511ad001a777f8e/issues/us-epa-camd/easey-ui/2399)

**Comment by aprematta** [comment]
Testing complete, No isssues found
